django_stomp/builder.py

```diff
@@ -66,7 +66,7 @@ def _build_connection_parameter(
         "subscriptionId": subscription_id,
         "vhost": getattr(settings, "STOMP_SERVER_VHOST", None),
     }
-    extra_params = {"use_ssl": eval_str_as_boolean(settings.STOMP_USE_SSL), "client_id": client_id}
+    extra_params = {"client_id": client_id}
```

Comment:
**suggestion**: You can update the README to explain why `STOMP_USE_SSL` will remove and add a warning about deprecating this envvar

Reply:
This envvar will still be used, in the firsts implementations I had removed it, but it is already being used again